pkg/sql/parser/sql.y

```diff
@@ -8355,6 +8376,7 @@ unreserved_keyword:
 | DAY
 | DEALLOCATE
 | DELETE
+| DEFERRED
```

Comment:
Grammar changes including new keywords after RC1? I think this missed the cutoff and needs to wait for 2.2. Telemetry work really needs to be planned as a part of the main development cycle; it's not something we can do post-beta when we're only supposed to be making stability-improving changes. @awoods187

Reply:
This is an unreserved keyword. It doesn't change the set of queries that will be accepted nor the set of queries that produce errors.